SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -413,10 +467,33 @@ void USpatialSender::SendRPC(TSharedRef<FPendingRPCParams> Params)
 		// Target object was destroyed before the RPC could be (re)sent
 		return;
 	}
-
 	UObject* TargetObject = Params->TargetObject.Get();
+
+	if (AActor* TargetActor = Cast<AActor>(TargetObject))
+	{
+		USpatialNetConnection* SpatialConnection = NetDriver->GetSpatialOSNetConnection();
```

Comment:
You could call `NetDriver->GetSpatialOSNetConnection()` in place in `CreateActorChannel`.